pkgs/applications/networking/jmeter/default.nix

```diff
@@ -1,19 +1,37 @@
-{ fetchurl, stdenv, jre }:
+{ fetchurl, stdenv, jre, makeWrapper }:
 
 stdenv.mkDerivation rec {
-  name = "jmeter-3.3";
+  name = "jmeter-${version}";
+  version = "4.0";
   src = fetchurl {
     url = "http://archive.apache.org/dist/jmeter/binaries/apache-${name}.tgz";
-    sha256 = "190k6yrh5casadphkv4azp4nvf4wf2q85mrfysw67r9d96nb9kk5";
+    sha256 = "1dvngvi6j8qb6nmf5a3gpi5wxck4xisj41qkrj8sjwb1f8jq6nw4";
   };
 
-  buildInputs = [ jre ];
+  buildInputs = [ jre makeWrapper ];
+
+  phases = [ "unpackPhase" "installPhase" "checkPhase" "fixupPhase" ];
```

Comment:
Please use `dontBuild = true` instead.

Reply:
I also had to change the order of the `checkPhase` though